src/cassio/table/base_table.py

```diff
@@ -392,18 +397,24 @@ def _get_db_setup_cql(self, schema: Dict[str, List[ColumnSpecType]]) -> str:
         pk_spec = ", ".join(col for col, _ in schema["pk"])
         cc_spec = ", ".join(col for col, _ in schema["cc"])
         primkey_spec = f"( ( {pk_spec} ) {',' if schema['cc'] else ''} {cc_spec} )"
+
+        table_options = []
+
         if schema["cc"]:
             clu_core = ", ".join(
                 f"{col} {self.ordering_in_partition}" for col, _ in schema["cc"]
             )
-            clustering_spec = f"WITH CLUSTERING ORDER BY ({clu_core})"
+            table_options.append(f"CLUSTERING ORDER BY ({clu_core})")
+
+        if len(table_options) > 0:
+            options_clause = "WITH " + " AND ".join(table_options)
         else:
-            clustering_spec = ""
-        #
+            options_clause = ""
+
```

Comment:
i refactored this to accommodate adding the table comment. 

I figured I'd leave the new code (despite removing the comment property) so that adding additional options in the future is easier.